addons/website/views/snippets/snippets.xml

```diff
@@ -988,7 +988,7 @@
 
     <!-- Stretch section -->
     <div data-js="ContainerWidth" data-selector="section, .s_carousel .carousel-item, s_quotes_carousel .carousel-item"
-        data-target="> .container, > .container-fluid, > .o_container_small">
+        data-target="> .container, > .container-fluid, > .o_container_small" data-exclude=".row *,.col *">
```

Comment:
- Add data-exclude before data-target (more logical as it goes alongside data-selector)
- `.col *` targets a subset of `.row *` (as all col are in rows) so it would be useless, but `.row *` is not good as it target columns themselves, while you want to target "elements inside columns". Also "col" does not target all bootstrap columns (col-6, col-md-4, ...). The right selector is probably `.row > div > *` (elements which are children of children of rows (columns)).

Reply:
Ok, thanks ! After testing I think `.row > div *` is the appropriate one as we want to target every child of such div (or col)

Reply:
Make sure to test event description edition or something like that. I am afraid that `.row > div *` is too generic and will match "main top snippet" which are in complex layout using columns (which is an entire problem on its own but still...)

Reply:
After testing on the event description snippet, which is using a row to display the snippets, I see that ContainerWidth and ScrollButton do not work as they should when the snippets are displayed within a row, so maybe it is better to remove them in that case too. To me the only option that can be kept out of :
- ScrollButton
- Background
- MinHeight
- ContainerWidth
would be the MinHeight, that we could also keep on the contact us form page, as it is effective on it. What do you think ?
The thing is that `.row > div > *` will not suit the use case exposed in the task as the form from contact us page is not the direct child of the column

Reply:
> After testing on the event description, which is using a row to display the snippets, I see that ContainerWidth and ScrollButton do not work as they should when the snippets are displayed within a row

For ContainerWidth, it's not really true, it's not that is it not working, it is that is has no effect *in that particular* design. Any customization to the event page by a module or whatever could make it have an effect (and probably will in a future refactoring in standard you can be sure). What we could do is improve the option itself to not be shown when it has no effect (which is what we did at the beginning but then we ignored the fact that it could have an effect on mobile or whatever). So my guess is that we can leave it there for now.

For ScrollButton, that's just a bug that should be fixed. The option makes sense there and should work. You can fix this in this PR in another commit if possible.

> The thing is that .row > div > * will not suit the use case exposed in the task as the form from contact us page is not the direct child of the column

That won't be true anymore in the current refactoring being done for contact us. But actually there may be an easier solution, instead of `.row > div *` or `.row > div > *`:
- `[data-snippet] .row > div *` -> "elements which are descendants of a column which is in a snippet"

Reply:
I will add in the description of the task that the expected behavior on the contact us form will be testable once https://github.com/odoo/odoo/pull/69888 is merged. With that `[data-snippet] .row > div *` is exactly what we need, and I will fix the ScrollButton issue in a separate commit :+1:

Reply:
I opened a PR for the ScrollButton fix here https://github.com/odoo/odoo/pull/72647